coverage.py

```diff
@@ -1373,6 +1373,23 @@ def process_item(self, work):
         return work
 
 
+class MARCRecordWorkCoverageProvider(WorkPresentationProvider):
+    """Make sure all presentation-ready works have an up-to-date MARC
+    record.
+
+    This is different from the MARCRecordCacheMonitor, which sweeps
+    over all presentation-ready works, even ones which are already
+    covered.
```

Comment:
This was based on the OPDS code, I may not actually need both of the monitor and the coverage provider.

Reply:
I agree; I think we don't need the monitor. IMO Our pattern where we strategically remove coverage when we want the work to be done again makes these monitors unnecessary.

Reply:
Now that the monitor is gone, remove the second paragraph here.